Makefile

```diff
@@ -1,48 +1,116 @@
 #! /usr/bin/make -f
+
 SHELL:=/bin/bash
 
+define pod
+
+=pod
+
+=head1 SYNOPSIS
+
+  make [-n] key=I<trello_key> token=I<trello_token> board=I<board_id> moment=I<moment_name> <command>
+  where command is one of: import ${board}.json thumbnails  ${board}.ttl moment
+
+This Makefile is used create a dams moment from a trello board.  The commands allow for the creation of a number of items needed for the momment
+
+=head1 COMMANDS
+
+=over 4
+
+=item B<make import>
+
+Imports all the required data from the trello board.  This
+
+=back
+
+=head1 OPTIONS
+
+=over 4
+
+=item B<key=I<trello_key>>
+
+This is the key for API access to the trello key. You can get this from....
+
+=item B<token=I<trello_token>>
+
+This is the token for API access to the trello key. You can get this from....
+
+=back
+
+=head1 FILE STRUCTURE
+
+The following diagram shows an example file structure.
+
+- moments
+		- cats
+    - cats_moment.ttl (the moment description)
+			- cats.ttl (new part)
+	- pY20Yz5x.json (This is the downloaded board)
+	- pY20Yz5z.ttl ( This is created from trello to moment )
+	- cats
+		- cats.json ( converted from ../pY20Yz5z.ttl with RIOT )
+			- Z4444 (a card)
+					- image_name.jpg
+
+=cut
+
+endef
```

Comment:
This is a method for creating documentation in tools like Makefiles, or  things that don't have other methods.  You can look at https://perldoc.perl.org/perlpod.html for more information